frontend/tests/unit/store.projectQuota.spec.js

```diff
@@ -0,0 +1,93 @@
+//
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import { getProjectQuotaStatus, aggregateResourceQuotaStatus } from '@/store/modules/projectQuota'
```

Comment:
I would not export these two helper function from the store and instead would add the aggregation to the mutation and call it `SET_RESOURCE_QUOTAS`:

```js
// actions
const actions = {
  async fetchProjectQuota ({ commit }, namespace) {
    const { data } = await getResourceQuotas({ namespace })
    commit('SET_RESOURCE_QUOTAS', [namespace, data])
  }
}

// mutations
const mutations = {
  SET_RESOURCE_QUOTAS (state, [key, quotas]) {
    const value = aggregateResourceQuotaStatus(map(quotas, 'status'))
    Vue.set(state, key, value)
  }
}
```

The tests can remain the same if we defined the two function locally by calling the getter and mutation:
```js
import projectQuota from '@/store/modules/projectQuota'

function getProjectQuotaStatus (quota) {
  const namespace = 'default'
  return projectQuota.getters.status({ [namespace]: quota }, null, { namespace })
}

function aggregateResourceQuotaStatus (statuses) {
  const namespace = 'default'
  const state = {}
  const quotas = statuses.map(status => ({ status }))
  projectQuota.mutations.SET_RESOURCE_QUOTAS(state, [namespace, quotas])
  return state[namespace]
}
```

Reply:
ok.